draft-rescorla-tls-esni.md

```diff
@@ -193,10 +193,23 @@ The cipher suite list is orthogonal to the
 list of keys, so each key may be used with any cipher suite.
 
 This structure is placed in the RRData section of a TXT record as
-encoded above. The Resource Record TTL determines the lifetime of
-the published ESNI keys. Clients MUST NOT use ESNI keys beyond
-their recommended lifetime. Note that the length of this structure
-MUST NOT exceed 2^16 - 1, as the RDLENGTH is only 16 bits {{RFC1035}}.
+encoded above. The name of each TXT record MUST match the name composed
+of "_esni" and the query domain name. That is, if a client queries 
+example.com, the ESNI TXT name is _esni.example.com.
+Servers SHOULD configure DNS such that, upon querying a domain name
+with ESNI support, at most one A, AAAA, TXT ESNI, and ALTSVC {{?I-D.schwartz-httpbis-dns-alt-svc}} 
+Resource Record is returned. Providing more address options and ESNI keys could lead
```

Comment:
I'm having trouble following this. Is this each record?

Reply:
Missed period.

Reply:
Indeed, the sentence is confusing. I just removed it. The point I was trying to make is that if there are multiple IP addresses to choose from with multiple keys, and the server does not configure each of those addresses with the set of keys, the client might pick the wrong address/key combo.